app/Core/Template.php

```diff
@@ -100,44 +132,35 @@ class Template
      * @access public
      */
     public function __construct(
-        /** @var Theme */
-        private Theme $theme,
-
-        /** @var Language */
-        public Language $language,
-
-        /** @var Frontcontroller */
-        public Frontcontroller $frontcontroller,
-
-        /** @var IncomingRequest */
-        public IncomingRequest $incomingRequest,
-
-        /** @var Environment */
-        private Environment $config,
-
-        /** @var AppSettings */
-        private AppSettings $settings,
-
-        /** @var AuthService */
-        private AuthService $login,
-
-        /** @var Roles */
-        private Roles $roles,
-
-        /** @var \Illuminate\View\Factory|null */
-        public ?Factory $viewFactory = null,
-
-        /** @var CompilerInterface|null */
-        private ?CompilerInterface $bladeCompiler = null,
```

Comment:
I cleaned all this up before, why did we undo it?

Reply:
not entirely sure, honestly. Reverting